api/admin.go

```diff
@@ -399,9 +399,33 @@ func getAnalytics(c *Context, w http.ResponseWriter, r *http.Request) {
 			rows[4].Value = float64(r.Data.(int64))
 		}
 
-		rows[5].Value = float64(TotalWebsocketConnections())
-		rows[6].Value = float64(Srv.Store.TotalMasterDbConnections())
-		rows[7].Value = float64(Srv.Store.TotalReadDbConnections())
+		// If in HA mode then aggregrate all the stats
+		if einterfaces.GetClusterInterface() != nil && *utils.Cfg.ClusterSettings.Enable {
+			stats, err := einterfaces.GetClusterInterface().GetClusterStats()
+			if err != nil {
+				c.Err = err
+				return
+			}
+
+			totalSockets := 0
+			totalMasterDb := 0
+			totalReadDb := 0
+
+			for _, stat := range stats {
+				totalSockets = totalSockets + stat.TotalWebsocketConnections
+				totalMasterDb = totalMasterDb + stat.TotalMasterDbConnections
+				totalReadDb = totalReadDb + stat.TotalReadDbConnections
+			}
```

Comment:
Unless I'm reading this wrong, doesn't this miss adding the stats for the current server?